src/puppet-dom.js

```diff
@@ -24,8 +24,8 @@
       this.listening = true;
 
       this.element.addEventListener('click', clickHandler);
-      window.addEventListener('popstate', this.historyHandler); //better here than in constructor, because Chrome triggers popstate on page load
-      this.element.addEventListener('puppet-redirect-pushstate', this.historyHandler);
+      window.addEventListener('popstate', historyHandler); //better here than in constructor, because Chrome triggers popstate on page load
```

Comment:
I'd rather do `self.historyHandler` for extendability feature, so someone would be able to change the `#historyHandler` for given instance, not for entire prototype.
If we are not planning such extendability I would remove `#historyHandler` from prototype, and move it as private method to the constructor.

Aren't you ding the same for `unlisten`?